packages/scaffold-config/src/frameworks.ts

```diff
@@ -86,7 +86,6 @@ export const WIZARD_FRAMEWORKS = [
     dependencies: (bundler: WizardBundler['type'], projectPath: string): Promise<DependencyToInstall[]> => {
       return Promise.all([
         isDependencyInstalled(dependencies.WIZARD_DEPENDENCY_REACT_SCRIPTS, projectPath),
-        isDependencyInstalled(dependencies.WIZARD_DEPENDENCY_WEBPACK, projectPath),
```

Comment:
`webpack` is a transient dependency of `react-scripts` so we don't need to list it here, same for `vue-cli`